src/components/pages/ClassInfoPage.jsx

```diff
@@ -0,0 +1,227 @@
+        } finally {
+          setIsLoading(false);
+        }
+      };
+
+      fetchClassAndTeacher();
+    } else {
+      console.error('No token found, user might not be logged in.');
+    }
+  }, [classId, userData]);
+
+  if (isLoading) return <Loader />;
+  if (error) return <p>Error: {error}</p>;
+  if (!classItem) return <p>Class not found.</p>;
+
+  return (
+    <div className="container m-auto p-10">
+      <div className="flex flex-col w-full text-black">
+        <div className="flex justify-between flex-col lg:flex-row">
+          <div className="flex flex-col w-full lg:w-[75%] md:flex-row lg:flex-row pb-4">
+            <div className="w-full m:w-[60%] font-roboto text-xl">
+              <h2 className="font-spartan font-medium text-4xl pb-8">
+                {classItem.classTitle}
+              </h2>
+              <p className="w-full md:w-[95%]">{classItem.description}</p>
+            </div>
+
+            <div className="flex w-full lg:w-3/4 pl-0 md:pl-2">
+              <img
+                className="rounded-lg"
+                src={classItem.classImageUrl}
+                alt={classItem.classTitle}
+              />
+            </div>
+          </div>
+
+          <div className="flex flex-col w-full lg:w-[15%] text-black   rounded-xl mt-4 mb-8 lg:m-4">
+            <div>
+              <div className="border-2 border-lightGreen rounded-lg flex flex-row flex-wrap justify-between lg:flex-col">
+                <div className="flex flex-col text-center items-center justify-center text-black p-4 mb-4 lg:bg-lightGreen">
+                  <p className="flex text-3xl items-center font-bold">
+                    ${classItem.price}
+                  </p>
+                  <p className="flex text-xl items-center font-semibold">
+                    per session
+                  </p>
+                </div>
+
+                <div className="flex items-center p-2">
+                  <img src={AgeIcon} alt="Icon age" className="w-6 h-6" />
+                  <span>
+                    Ages: {classItem.ages.minAge} - {classItem.ages.maxAge}
+                  </span>
+                </div>
+                <div className="flex items-center p-2 gap-2">
+                  <img src={IconClock} alt="Icon clock" className="w-6 h-6" />
+                  <span>{classItem.duration} min</span>
+                </div>
+                <div className="flex items-center p-2 gap-2">
+                  <img
+                    src={LessonTypeIcon}
+                    alt="Icon age"
+                    className="w-6 h-6"
+                  />
+                  <span>{classItem.type}</span>
+                </div>
+                <div className="flex items-center p-2 gap-2 md:flex">
+                  <img
+                    src={ScheduleIcon}
+                    alt="Schedule Icon"
+                    className="w-6 h-6"
+                  />
+                  <span>On Request</span>
+                </div>
```

Comment:
Could you please add info about future lesson type here. (lessonType: Group or 1:1)